common_features.mk

```diff
@@ -267,20 +267,20 @@ ifeq ($(strip $(ENCODER_ENABLE)), yes)
     OPT_DEFS += -DENCODER_ENABLE
 endif
 
-ifeq ($(strip $(HAPTIC_ENABLE)), DRV2605L)
-    COMMON_VPATH += $(DRIVER_PATH)/haptic
-    SRC += haptic.c
+ifneq ($(strip $(HAPTIC_ENABLE)),)
```

Comment:
Wouldn't this be true if `HAPTIC_ENABLE = no`?

Reply:
ah, yeah, it would be.  

Would this be better: 
```suggestion
HAPTIC_ENABLE ?= no
ifneq ($(strip $(HAPTIC_ENABLE)),no)
```

Reply:
If we're anticipating more than just `DRV2605L` and `SOLENOID` as haptic drivers (is `yes` a valid option?), it might be worth it.

Reply:
I don't think that we plan any more than just these two.  I mean, more is possible, but the issue would be finding the haptic driver for it. 

Honestly, I wasn't really 100% sure of the best way to handle this.

Reply:
In that case, the above change LGTM :)